src/helpers/truncateAddress.js

```diff
@@ -1,4 +1,5 @@
 export const truncateAddress = (address, digits) => {
+  if (!address) return
```

Comment:
for the next time, please add systematically adds a `TODO` to indicate that the code needs to be reviewed.
Because iddealy we must return an Error object.